src/api/handlers/guild.ts

```diff
@@ -769,13 +769,9 @@ export function getGuild(guildID: string, counts = true) {
 }
 
 /** Returns the guild template if it exists */
-export function getGuildTemplate(
-  guildID: string,
-  templateCode: string,
-) {
-  return RequestManager.get(
-    `${endpoints.GUILD_TEMPLATES(guildID)}/${templateCode}`,
-  ) as Promise<Template>;
+export async function getTemplate(templateCode: string) {
```

Comment:
This now makes this a breaking change. I would recommend adding this as separate function. Marking the old one as @deprecated and then make a new pr in v11 thats going to remove this old function

Reply:
Well ayntee did not want that

Reply:
@itohatweb 
![image](https://user-images.githubusercontent.com/34645569/105388764-7c634c00-5c30-11eb-9004-616fc1576fda.png)